tutorials/admin/users/users.md

```diff
@@ -1 +1,399 @@
+
+#### stac-admin
+
+Members of this group will have full read and write permissions to the [STAC catalog](../../users/catalog/catalog.md).
+Add users to this group who will need to update or manage the catalog.
+
+#### monitoring
+
+Members of this group will have full access to the Grafana and Prometheus [monitoring](../monitoring/monitoring.md) 
+components.
+
+This group is useful if you want to allow a user to be able to view sensitive monitoring data about your node without
+giving them full administrator permissions.
+
+(service-permissions)=
+## Service Permissions
+
+Marble [services](../services/services.md) are user facing tools that are deployed with the Marble software stack.
+Because they are user facing, you will likely want to configure which services are available to which users.
+
+As a first step you will want to decide which resources are accessible by users with accounts on your node (those in 
+the [users](users-group) group), and which will by publicly available (as defined by the [anonymous](anonymous-group) 
+group).
+
+Then you can add more groups and permission rules as you see fit. Some examples:
+
+- you want to host and serve some data that requires users to agree to terms and conditions before they get access.
+- you only want to provide access to the IDE to some users but not others.
+- you only want to allow some users to be able to create new 
+  [remote processing](../../users/remote-processing/remote-processing.md) workflows.
+
+### Managing Service Permissions
+
+To edit service permissions with Magpie, click the "Edit Services" button:
+
+```{image} images/magpie-edit-services-button.png
+:alt: Magpie Edit Services Button
+:width: 100px
+:align: center
+```
+
+This page shows each of the available service types:
+
+```{image} images/magpie-service-types.png
+:alt: Magpie Service Types
+:align: center
+```
+
+Here we will discuss the service types that are the most relevant for the Marble software. For a full description of all
+service types available through Magpie, see the 
+[Magpie documentation](https://pavics-magpie.readthedocs.io/en/latest/services.html#available-services).
+
+(service-permissions-api)=
+#### api
+
+This service type allows you to define access rules based on URL path segments.
+
+For example, the `STAC` service has the following endpoints:
+
+- stac/
+- stac/collections
+- stac/search
+
+And you can [define an access rule](permissions-child-resource) for each!
+
+Permissions for api services can either be:
+
+- `read`: allow/deny users to make GET and HEAD requests to the endpoint
+- `write`: allow/deny users to make all other requests methods to the endpoint
+
+#### geoserver
+
+This service type is used to apply permissions for the [Geoserver](https://geoserver.org/) service.
+
+This service type will only be used if the [included geoserver component is enabled](../services/services.md).
```

Comment:
Maybe I recall incorrectly, but I don't think the version of services tutorial you have mentioned geoserver or any other services by name.

Reply:
No, not yet at least until I can get the birdhouse documentation up to date.
Do you think I should remove the link until then?

Reply:
I leave that to you. I am totally good with leaving it here if that helps you keep track of this.